tests/bsim/bluetooth/host/adv/resume2/connecter/main.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/bluetooth/hci_types.h"
+#include <errno.h>
+#include <testlib/conn.h>
+#include <testlib/scan.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/__assert.h>
+
+LOG_MODULE_REGISTER(connecter, LOG_LEVEL_INF);
+
+/* @file
+ *
+ * This app scans for a device with the name "dut" and connects
+ * to it. It then waits for the connection to be disconnected,
+ * before starting over.
+ *
+ * This app is added to the simulation simply to excerise the
+ * the DUT's connectable advertiser.
+ *
+ * Mulitple instances of this app are added to the simulation,
+ * to excericse BT_MAX_CONN of the DUT.
```

Comment:
you need to exorcise this word (twice lol)

Reply:
I have now invested in a spell checker 🧠